zlp-scheduler/app/views/student/view_terms.html.haml

```diff
@@ -2,7 +2,7 @@
 
 %div{:class => "term_frame"}
     %div{:class => "header_div"}
-        %h2{:class => 'term_header'} Fall 2020
+        %h2{:class => 'term_header'} #{(Term.find_by active: 1).name}
```

Comment:
It might be better to have this code be a variable from the controller that you refer to here instead of executing the query here in the View.